dashboard/app/controllers/foorm_form_editor_controller.rb

```diff
@@ -1,10 +1,10 @@
 # Foorm Editor is only available on levelbuilder or test, for those with levelbuilder permissions.
-class FoormEditorController < ApplicationController
+class FoormFormEditorController < ApplicationController
   before_action :require_levelbuilder_mode_or_test_env
   before_action :authenticate_user!
   authorize_resource class: false
 
-  # GET '/foorm/editor/'
+  # GET '/foorm/editor/forms'
```

Comment:
nit: `foorm/form/editor` makes more sense to me, since it is the foorm::form editor